subsys/bluetooth/services/ots/ots_oacp.c

```diff
@@ -318,7 +318,7 @@ static void oacp_read_proc_cb(struct bt_gatt_ots_l2cap *l2cap_ctx,
 	}
 
 	if (len < 0) {
-		LOG_ERR("OCAP Read Op failed with error: %d", len);
+		LOG_ERR("OCAP Read Op failed with error: %zu", len);
```

Comment:
`len` is a `ssize_t` and not a `size_t`, so I guess this should be 
```suggestion
		LOG_ERR("OCAP Read Op failed with error: %zd", len);
```
instead, right?

Reply:
I used zu because that is what was used for output of the length in the original PR.
We should at least be consistent.

Reply:
Can a size_t be negative?

Reply:
`ssize_t` is signed, `size_t` is unsigned. So no.

Reply:
"size_t is an unsigned integer type"
https://en.wikipedia.org/wiki/C_data_types#stddef.h

Reply:
Ah - now I have properly read your comment 
No, len is a size_t.  The function return a ssize_t.

Reply:
@asbjornsabo `len` is defined as: `ssize_t len;` :)

Reply:
You guys are absolutely right - and I am wrong.  (I was so hung up in another fix I had to do related to this that I kept staring me blind on another len parameter ...)
Fixed now.